docker/Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM docker.montagu.dide.ic.ac.uk:5000/orderly.server:51680f5
```

Comment:
If you can deal with https://vimc.myjetbrains.com/youtrack/issue/VIMC-1907 then update this reference the build should start working